app/api/dao/user.py

```diff
@@ -143,7 +143,7 @@ def get_user_by_username(username: str):
         return UserModel.find_by_username(username)
 
     @staticmethod
-    def list_users(user_id: int, search_query: str = "", is_verified = None):
+    def list_users(user_id: int, search_query: str = "", page: int = 0, size: int = 10, is_verified = None):
         """ Retrieves a list of verified users with the specified ID.
         
         Arguments:
```

Comment:
It seems you forgot to update the doc with the new params

Reply:
Whoops, good catch! Fixed.